modules/tinymce/src/models/dom/main/ts/table/selection/Ephemera.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) Tiny Technologies, Inc. All rights reserved.
+ * Licensed under the LGPL or a commercial license.
+ * For LGPL see License.txt in the project root for license information.
+ * For commercial licenses see https://www.tiny.cloud/
+ */
+
+import { Ephemera as DarwinEphemera } from '@ephox/darwin';
```

Comment:
For these duplicated things, I know we still need to solve that. However, to make sure changes are made across all of them can we please add comments at the top of the duplicated files to make it clear they are duplicated and where they are duplicated?

- [ ] Add comments to duplicated files to say where they are duplicated

Reply:
That's a good idea. Thanks for the suggestion. I have added a note to all of the duplicated and partially duplicated files: Events, Ephemera, TableTargets, TableSize, TableSelection, Utils